config_src/mct_driver/mom_surface_forcing_mct.F90

```diff
@@ -237,9 +237,11 @@ subroutine convert_IOB_to_fluxes(IOB, fluxes, index_bounds, Time, G, US, CS, &
                               !! is present, or false (no restoring) otherwise.
   logical :: restore_sst      !< local copy of the argument restore_temp, if it
                               !! is present, or false (no restoring) otherwise.
-  real :: delta_sss           !< temporary storage for sss diff from restoring value
+  real :: delta_sss           !< temporary storage for sss diff from restoring value2
```

Comment:
Was this "2" intentional? Just curious...

Reply:
No, that "2" at the end of a comment was a mistake. Sorry about that.  Although I tried to be as careful as possible, it would be helpful if someone from NCAR (@gustavo-marques ?) would please evaluate whether the few necessary changes to the mct_driver and nuopc_driver code are as benign as they were intended to be.

Reply:
@Hallberg-NOAA Do you want to wait for feedback from MCT/NUOPC devs before merging?